packages/snap-toolbox/src/getContext/getContext.ts

```diff
@@ -33,8 +33,15 @@ export function getContext(evaluate: string[] = [], script?: HTMLScriptElement |
 		throw new Error('getContext: first parameter must be an array of strings');
 	}
 
+	let siteIdString = 'siteId';
+
 	const variables: ContextVariables = {};
 
+	//add siteId to list if its not already in there
+	if (evaluate.indexOf(siteIdString) == -1) {
+		evaluate.push(siteIdString);
+	}
+
```

Comment:
This should probably be opt-in only - and modified in snap-preact to request it.

Reply:
I was just thinking about this a bit more... how do we get the siteId when doing local development? The script src is typically `https://localhost:3333/bundle.js` in these cases... I guess they could add the siteId to the script context?